build-system/tasks/storybook/amp-env/webpack.config.js

```diff
@@ -20,12 +20,14 @@ module.exports = ({config}) => {
     modules: [
       path.join(__dirname, '../node_modules'),
       path.join(__dirname, '../../../../node_modules'),
+      path.join(__dirname, '../../../..'),
```

Comment:
FMI: Does the Webpack config need to be updated somehow? Can Storybook still get files from `src/`?